deps/build.jl

```diff
@@ -1,15 +1,48 @@
 #######################################################################
-import JSON
+import JSON, Conda
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
 # Make sure Python uses UTF-8 output for Unicode paths
 ENV["PYTHONIOENCODING"] = "UTF-8"
 
-include("jupyter.jl")
-const jupyter, jupyter_vers = find_jupyter()
+
+const jupyter, jupyter_vers = try
+    "jupyter",convert(VersionNumber, chomp(readall(`jupyter-kernelspec --version`)))
+catch e1
+    try
+        "ipython",convert(VersionNumber, chomp(readall(`ipython --version`)))
```

Comment:
If we intend to use `jupyter kernelspec` in the future when https://github.com/jupyter/notebook/issues/448 is fixed we should do:
```
function prog_version(prog, arg)
    try
       return convert(VersionNumber, chomp(readall(`$prog $arg --version`)))
    catch
       return v"0.0"
    end
end

jupyter = ""
jupyter_vers = v"0.0"
for (p arg) in (("jupyter", `kernelspec`),("ipython", ``),("ipython2", ``),("ipython3", ``),("ipython.bat", ``)) # for now ("jupyter", `kernelspec`) should be ("jupyter-kernelspec", ``) due to issue
    v = prog_vers(p, arg)
    if v >= v"3.0"
       jupyter = p
       jupyter_vers = v
       break
    end
end